LDAR_Sim/src/file_processing/output_processing/summary_outputs.py

```diff
@@ -105,17 +110,21 @@ def generate_emissions_summary(directory: str, outputs_mapper: SummaryOutputMapp
         file_processing_const.Multi_Sim_Output_Const.EMIS_PATTERN,
     )
     estimated_df: pd.DataFrame = generate_emissions_estimation_summary(directory, outputs_mapper)
+    # Merge summary and estimated data
     merged = pd.merge(
         emissions_summary_df,
         estimated_df,
-        on=[
-            output_file_constants.SummaryFileColumns.CommonColumns.PROGRAM_NAME,
-            output_file_constants.SummaryFileColumns.CommonColumns.SIMULATION_NUMBER,
-        ],
+        on=common_columns,
         how="outer",
     )
-    # Fill NaN values with zeros
-    merged.fillna(0, inplace=True)
+
+    # Exclude specific columns from being converted to numeric
+    cols_to_convert = merged.columns.difference(common_columns)
```

Comment:
Are there any other columns that are supposed to be non numeric?

Reply:
Nope. All the rest are summary columns based on emissions numbers.